### DISCUSSION TITLE
Threshold and sowing date

### DISCUSSION THREAD

**Discussion by Topex22** [roe-dl/weewx-GTS#18] [Q&A]
Hello,

First, thank you so much for your work.

1. yearGDD and seasonGDD shows the values ​​with a threshold value of 10°C wether I have well understood. What is the way to turn into 6°C to deal with grasslands ?

2. What is the way to initiate the calculation of sums from a sowing date ?

Thanks to your lights

**Comment by roe-dl** [comment]
There is a parameter in `weewx.conf` 
```
[StdReport] 
    [[Defaults]] 
        [[[Units]]] 
            [[[[DegreeDays]]]] 
                growing_base = 50, degree_F
```

If no other value is specified that value is used. For other base temperatures you can specify it, for example:
```
$year.outTemp.GDD({'base':(6.0,'degree_C')})
```

To start at an arbitrary date you could write (not tested):
```
#from weewx.units import ValueTuple, ValueHelper 
#set $timespan=...
#set $gdd=0
#for $dd in $timespan.days
#set $nowgdd=$dd.outTemp.GDD({'base':(6.0,'degree_C')}).degree_C_day.raw
#if $nowgdd is not None
#set $gdd+=$nowgdd
#end if
#end for
#set gdd_vh=ValueHelper(ValueTuple($gdd,'degree_C_day','group_degree_day'),formatter=$station.formatter)
$gdd_vh
```
The timespan needs to be of type `TimespanBinder`.

**Reply by Topex22** [reply]
I have modify weewx.conf.

But I have not understood how use your codes. I use the Belchertown skin, and I hoped that it was in graph.conf that I needed to work about my sowing date.

http://chevrerie-du-cap.com/meteo/

**Comment by roe-dl** [comment]
Ah, ok, it's about diagrams. The examples I gave you can use in `index_hook_after_*.inc` files. I have to look how to express that in `graphs.conf`. That will take a while.

**Reply by Topex22** [reply]
thank you
